teos/src/main.rs

```diff
@@ -176,7 +177,14 @@ async fn main() {
     };
     log::info!("Last known block: {}", tip.header.block_hash());
 
-    let mut poller = ChainPoller::new(&mut derefed, Network::from_str(&conf.btc_network).unwrap());
+    // This is how chain poller names bitcoin networks.
+    let btc_network = match conf.btc_network.as_str() {
+        "main" => "bitcoin",
+        "test" => "testnet",
+        any => any,
```

Comment:
It is a bit annoying that we have to cast it back and forth, but I guess it is how it is until we can simplify the codebase to use a single "bitcoind rpc"

Reply:
Gonna leave this here so we can improve it if it gets merged:

https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/247